Discussion Forum Post UI + Routing

stephen8chang opened the pull request:
WIP, only look at commit 48be1f7 onwards

mayankshouche reviewed (commented):
The new component looks really good!

I was wondering if you intentionally changed the comments section, cause I'm getting a 'reply' dropdown under all the comments now, including second level ones: 
![image](https://user-images.githubusercontent.com/43075711/115306082-eb1b4d00-a12c-11eb-9f76-fd79a1b72a11.png)

Maybe also go ahead and get rid of the sample component on the Comment view, and have it start rendering on the Discussion Forum page.

stephen8chang commented:
> I was wondering if you intentionally changed the comments section, cause I'm getting a 'reply' dropdown under all the comments now, including second level ones:

Fixed, try it out again and let me know if the issue persists 

> Maybe also go ahead and get rid of the sample component on the Comment view, and have it start rendering on the Discussion Forum page.

Temporarily moved to the Discussion Forum page

> I'm also getting one of the old errors from before, might be a merge error?

Fixed
